modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/server/persistence/RocksDbKeyValueStorage.java

```diff
@@ -239,20 +243,25 @@ public void start() {
     }
 
     private static List<ColumnFamilyDescriptor> cfDescriptors() {
-        Options dataOptions = new Options().setCreateIfMissing(true)
+        Options sharedOptions = new Options()
+                .setCreateIfMissing(true)
+                .setNumLevels(4)
```

Comment:
I think it will be useful to comment the options that were chosen and write a justification for the specific options and values chosen; otherwise, in the future, it will become difficult to remember why the configuration is as it is now.

I'm not sure whether we need to make these configurable by the user. Maybe we should leave the values hard-coded for now as they seem to be very specific to our Meta-Storage scenarios (and the load profile will not necessarily change from cluster to cluster).

Reply:
Done